src/validationinterface.cpp

```diff
@@ -62,10 +64,39 @@ void UnregisterAllValidationInterfaces() {
     g_signals.UpdatedTransaction.disconnect_all_slots();
     g_signals.EraseTransaction.disconnect_all_slots();
     g_signals.SyncTransaction.disconnect_all_slots();
+    g_signals.GetBatchScanner.disconnect_all_slots();
     g_signals.UpdatedBlockTip.disconnect_all_slots();
 }
 
-void SyncWithWallets(const CTransaction &tx, const CBlock *pblock, const int nHeight) {
+void AddTxToBatches(
+    std::vector<BatchScanner*> &batchScanners,
+    const CTransaction &tx,
+    const uint256 &blockTag,
```

Comment:
In all the functions in `wallet_scanner.rs` that take a block tag argument, that argument comes before the tx information. Non-blocking suggestion: do the same in the C++ code for consistency.

Reply:
Oh, pblock came after tx already in `SyncWithWallets`. Ok I guess this order makes sense as-is, then.